services/business/istio_validations.go

```diff
@@ -55,14 +54,7 @@ func (in *IstioValidationsService) GetNamespaceValidations(namespace string) (mo
 		return nil, err
 	}
 
-	pods, err := in.k8s.GetNamespacePods(namespace)
-	if err != nil {
-		log.Warningf("Cannot get pods for namespace %v.", namespace)
-		return nil, err
-	}
-
 	objectCheckers := []ObjectChecker{
-		checkers.RouteRuleChecker{Namespace: namespace, PodList: pods.Items, RouteRules: istioDetails.RouteRules},
```

Comment:
Not from this PR, but should we add checkers.PodChecker in this list? As we have it in L#38. In case of yes, I volunteer to add a new JIRA to support that.

Reply:
Yes, PodChecker could be added in a separate PR, I guess it was only used in the Graph.
In the ServiceDetails was never shown, so, it is a new addition.